### ISSUE TITLE
False-positive in `no-unnecessary-combination`

### ISSUE THREAD

**Issue by igorkamyshev** [effector/eslint-plugin#63]
As I understand this code:

```ts
sample({
  clock: combine({
    cashbackType: $cashbackType,
    savedTransferSelected: $selectedSavedTransfer,
  }),
  source: $estimatesDays,
  fn: (source, { cashbackType }) => mapper(source, cashbackType),
  target: cashbackEstimateDaysUpdated,
});
```

cannot be replaced with 

```ts
sample({
  clock: {
    cashbackType: $cashbackType,
    savedTransferSelected: $selectedSavedTransfer,
  },
  source: $estimatesDays,
  fn: (source, { cashbackType }) => mapper(source, cashbackType),
  target: cashbackEstimateDaysUpdated,
});
```

But rule throws warning now.
